lm/lm_image.py

```diff
@@ -0,0 +1,87 @@
+__all__ = ["LmImageJob"]
+
+import shutil
+
+from sisyphus import *
+from i6_core import rasr
+from i6_core import util
+
+Path = setup_path(__package__)
+
+
+class LmImageJob(rasr.RasrCommand, Job):
+    """
+    pre-compute LM image without generating global cache
+    """
+
+    def __init__(
+        self,
+        crp,
+        extra_config=None,
+        extra_post_config=None,
+        encoding="utf-8",
+        mem=2,
+    ):
+        kwargs = locals()
+        del kwargs["self"]
+
+        self.config, self.post_config = LmImageJob.create_config(**kwargs)
+        self.exe = self.select_exe(crp.lm_util_exe, "lm-util")
+
+        self.log_file = self.log_file_output_path("lm_image", crp, False)
+        self.out_image = self.output_path("lm.image", cached=True)
+
+        self.rqmt = {"time": 1, "cpu": 1, "mem": mem}
+
+    def tasks(self):
+        yield Task("create_files", resume="create_files", mini_task=True)
+        yield Task("run", resume="run", rqmt=self.rqmt)
+
+    def create_files(self):
+        self.write_config(self.config, self.post_config, "lm_image.config")
+        extra_code = (
+            ":${{THEANO_FLAGS:="
+            "}}\n"
+            'export THEANO_FLAGS="$THEANO_FLAGS,device={0},force_device=True"\n'
+            'export TF_DEVICE="{0}"'.format(
+                "gpu" if self.rqmt.get("gpu", 0) > 0 else "cpu"
+            )
+        )
+        self.write_run_script(self.exe, "lm_image.config", extra_code=extra_code)
+
+    def run(self):
+        self.run_script(1, self.log_file)
+        shutil.move("lm.image.1", self.out_image.get_path())
```

Comment:
please verify that this is (always) the correct name according to the config

Reply:
then I suggest to name this one 
```suggestion
        shutil.move("lm.image", self.out_image.get_path())
```

and below in the `create_config` method, after asserting there is only exactly one arpa lm, we explicitly overwrite `lm-util.lm.image = "lm.image` to make sure the name is correct.